ts/python/project.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it } from "https://deno.land/std@0.206.0/testing/bdd.ts";
+import * as garn from "./mod.ts";
+import { getStorePath, nixpkgsInput } from "../testUtils.ts";
+import { assert } from "https://deno.land/std@0.206.0/assert/mod.ts";
+import { renderFlakeFile } from "../nix.ts";
+import { formatFlake } from "../internal/runner.ts";
+import * as path from "https://deno.land/std/fs/copy.ts";
+import { existsSync } from "https://deno.land/std@0.201.0/fs/exists.ts";
+
+describe("mkPythonProject", () => {
+  it("produces correct store path structure", () => {
+    const tempDir = Deno.makeTempDirSync({ prefix: "garn-test" });
+    path.copySync("./examples/python-project/", `${tempDir}/python-project`);
```

Comment:
I do like that we have a python example directory, but would it be better to write out the python files here in this test-case without relying on files from `./examples`? So that the test is self-contained.

Reply:
Removed the ./example for now and inlined the files instead. Will create the examples again once I have a test for it.